tools/watermill-cli/README.md

```diff
@@ -0,0 +1,62 @@
+# CLI Wrapper for Watermill
+
+This is a CLI wrapper for Watermill. It has two basic functionalities: producing and consuming messages on the following
+Pub/Subs:
+
+1. Kafka
+2. Google Cloud Pub/Sub
+3. RabbitMQ
+
+## Consume mode
+
+In consume mode, the tool subscribes to a topic/queue/subscription (delete as appropriate) and prints the messages
+in a simplified format to the standard output:
+
+```
+[yyyy-mm-dd hh:mm:ss.ssssssss] topic: message payload
```

Comment:
I think that by default it should return raw data (pretty standard behaviour) plus information about flag to change output format

Reply:
done, now it just uses `PayloadMarshalFunc`

Reply:
b3ae9f252ccb6e90efcf9f72512765a9df71cff4